docs/configuration.md

```diff
@@ -59,6 +59,21 @@ search:
   button: false
 ```
 
+## Mermaid Diagrams
```

Comment:
Do you want to add a line here that links to the `docs/ui-components/code.md` page?

Reply:
Done.